frontend/actions.js

```diff
@@ -128,6 +130,42 @@ export default ({setState, getState}) => {
     })
   }
 
+  const checkAllFundsSendable = (state) => {
```

Comment:
this function should imho return a bool and it should be the responsibility of the calling function to set the state appropriately